sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -574,14 +642,15 @@ export class CertificateClient {
    * @summary Sets the certificate contacts.
    * @param options The optional parameters
    */
-  public async getCertificateContacts(options?: RequestOptionsBase): Promise<Contacts> {
-    const span = this.createSpan("getCertificateContacts", options);
+  public async getCertificateContacts(options: GetCertificateContactsOptions = {}): Promise<Contacts> {
+		const requestOptions = operationOptionsToRequestOptionsBase(options);
```

Comment:
nit: replace spaces with tabs (in a few places).

Reply:
I forgot to run the formatting tool. I usually don't think about formatting until I run prettier.